### PR TITLE
Ruby Group Capstone

### PR DESCRIPTION
**Pull request by PrantoshB**
# Hello There 👋🏽 

adsf

In this milestone, we have:
- Created class from UML diagram
- Created Unit Tests for every method using `RSpec`
- Created database using `SQL`
- Preserved data for each class in `JSON` files
- Fetch data from `JSON` files
- Fixed Linters Errors
- Followed Git and GitHub flows

## Here is the documentation [VIDEO](https://drive.google.com/file/d/1mVkJ-AenUX4Kt6WA8Ouy9SqBAf3xeVgI/view?usp=sharing)

### CODE REVIEW

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Good Teamwork 👍🏽
- Good App UI 👍🏽
- Data well preserved 👍🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
### Please ensure your project contains a video with a clear and objective description of the project in professional English. The topic and purpose of the project are made clear, and all the members of the team should appear in the video and talk about the project.

_Check the comments under the review._

## You can also consider:
### Adding a list of `genre` to your app to make it better 👍🏽 

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by cynthiainga — COMMENTED** [review]
Hi @PrantoshB, @DJ-MrJay & @gauravgangwarweb, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that all linters are failing. Please kindly fix your linters errors and then ask for review 🙏 🙏 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: menu.rb
Comment (PinkMoon25) [inline menu.rb:1]: - [ ] [OPTIONAL] Hi team, well done so far :clap:  , I know you have worked hard on this, I just wanted to suggest few things, 

1. so first the navigation can be simple with all the options present at once makes it easier for the user to use the app, currently you have to navigate here and there if you want to perform any action
2. while using the app I could not find an option to add label or genre or authors, I think you have to setup correct associations, like a book can have an author Item, this way we add an author with every book and so on...
3. for music albums even if I entered y or Y or yes it still makes the on_spotify attribute to false

**Review by pinkmoon25 — APPROVED** [review]
Hi @PrantoshB @gauravgangwarweb @DJ-MrJay , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
rubocop checks :heavy_check_mark: 
gitflow :heavy_check_mark: 
Pair-programming :heavy_check_mark: 
works on console :heavy_check_mark: 
Rspecs :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**